src/block_service/fds/fds_service.cpp

```diff
@@ -681,36 +651,10 @@ error_code fds_file_object::put_content(/*in-out*/ std::istream &is,
         return err;
     }
 
-    try {
-        // Get Object meta data
-        std::shared_ptr<galaxy::fds::FDSObjectMetadata> metadata =
-            c->getObjectMetadata(_service->get_bucket_name(), _fds_path);
-        const std::map<std::string, std::string> metaMap = metadata->metadata();
-
-        auto iter = metaMap.find(fds_service::FILE_MD5_KEY);
-        dassert(iter != metaMap.end(),
-                "can't find %s in object(%s)'s metadata",
-                fds_service::FILE_MD5_KEY.c_str(),
-                _fds_path.c_str());
-        _md5sum = iter->second;
-
-        // in a head http-request, the file length is in the x-xiaomi-meta-content-length
-        // while in a get http-request, the file length is in the contentLength
-        iter = metaMap.find(fds_service::FILE_LENGTH_CUSTOM_KEY);
-        dassert(iter != metaMap.end(),
-                "can't find %s in object(%s) metadata",
-                fds_service::FILE_LENGTH_CUSTOM_KEY.c_str(),
-                _fds_path.c_str());
-        _size = (uint64_t)atoll(iter->second.c_str());
-        transfered_bytes = _size;
-    } catch (const galaxy::fds::GalaxyFDSClientException &ex) {
-        derror("fds getObjectMetadata after put failed: remote_file(%s), code(%d), msg(%s)",
-               file_name().c_str(),
-               ex.code(),
-               ex.what());
+    ddebug("start to check meta data after successfully wrote data to fds");
+    if (get_file_meta() != ERR_OK) {
         err = ERR_FS_INTERNAL;
     }
-    FDS_EXCEPTION_HANDLE(err, "getObjectMetadata_after_put", file_name().c_str())
```

Comment:
`put_content` return value may be changed. In previous implementation, if the `Poco::TimeoutException` is caught up, the err will be `ERR_TIMEOUT`, but in current code, it will always be `ERR_FS_INTERNAL`. Is it a safe modification?